dqc_us_rules/dqc_us_0001.py

```diff
@@ -51,16 +53,18 @@ def filter_func(model_object):
                 )
 
             for axis in filter(filter_func, relset.fromModelObjects()):
-                _run_axis_checks(axis, axis_config, relset, val, role)
+                _run_axis_checks(axis, axis_key, axis_config, relset, val, role, checked_axes)
```

Comment:
Odd - didn't seem to work.  Just going to proceed with the key since that works for now.